buildSrc/src/main/resources/pom.xml

```diff
@@ -134,5 +134,13 @@
             <artifactId>micronaut-oraclecloud-atp</artifactId>
             <version>1.3.5</version>
         </dependency>
+
+        <!-- Oracle Cloud SDK Core -->
+        <dependency>
+            <groupId>com.oracle.oci.sdk</groupId>
+            <artifactId>oci-java-sdk-core</artifactId>
+            <version>1.34.0</version>
```

Comment:
Is this needed? It seems to be declared in the `micronaut-oraclecloud-bom` module which is dependency managed

Reply:
The pom.xml file isn't used as a pom file, it's parsed at https://github.com/micronaut-projects/micronaut-starter/blob/2.5.x/starter-core/src/main/java/io/micronaut/starter/build/dependencies/PomDependencyVersionResolver.java#L46

Reply:
Ignore the previous comment, you're right it isn't needed